catkin_home/src/navigation/base_control/launch/base_control.launch

```diff
@@ -0,0 +1,60 @@
+<launch>
+    <master 
+        auto="start"
+    />
+    <!-- Run the map server --> 
+    <arg 
+        name="map_file" 
+        default="$(find base_control)/maps/TMR_2019.yaml"
```

Comment:
please put the map file in [navigation>nav_main>maps](https://github.com/RoBorregos/Robocup-Home/tree/develop/catkin_home/src/navigation/nav_main/maps)